scripts/qemu-dm-wrapper

```diff
@@ -104,10 +109,21 @@ def main(argv):
     if os.path.exists("/usr/share/qemu-linaro/qemu"):
         qemu_args = qemu_args + ["-L", "/usr/share/qemu-linaro/qemu"]
 
+    if xenstore_exists("/local/domain/%d/platform/nic_type" % domid):
+        print "No explicit NIC selected. Defaulting to rtl8139. "
+    else:
+        nic = xenstore_read("/local/domain/%d/platform/nic_type" % domid)
```

Comment:
This logic should simply be:
nic_type = xenstore_read(...)
if nic_type:
    ...

which covers both the None and empty string case